packages/experimental/NativeButton/ios/MSFUIButtonViewManager.m

```diff
@@ -0,0 +1,34 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+#import "RCTBridgeModule.h"
+
+@import FluentUI;
+
+@implementation RCTConvert (MSFButtonAdditions)
+RCT_ENUM_CONVERTER(MSFButtonStyle, (@{
+	@"primary": @(MSFButtonStylePrimaryFilled),
+	@"secondary": @(MSFButtonStyleSecondaryOutline),
+	@"borderless": @(MSFButtonStyleBorderless),
+  @"acrylic": @(MSFButtonStylePrimaryFilled), // mac specific button style
+}), MSFButtonStylePrimaryFilled, integerValue);
+
+@end
+
+@interface RCT_EXTERN_MODULE(MSFUIButtonViewManager, RCTViewManager)
+
+//RCT_EXPORT_VIEW_PROPERTY(image, UIImage);
+RCT_EXPORT_VIEW_PROPERTY(onPress, RCTBubblingEventBlock);
+RCT_REMAP_VIEW_PROPERTY(buttonStyle, style, MSFButtonStyle);
+RCT_CUSTOM_VIEW_PROPERTY(title, String, MSFButton) {
+	// for simplicity only normal state is supported
+	[view setTitle:[RCTConvert NSString:json] forState:normal];
+}
+RCT_CUSTOM_VIEW_PROPERTY(isEnabled, BOOL, MSFButton) {
+	[view setEnabled:[RCTConvert BOOL:json]];
```

Comment:
Does this need to be custom because isEnabled != enabled?

Reply:
Good call. I was using custom view to set the enabled property. I think isEnabled didn't work because it's a swift only property. We should just export the enabled property so no need to make it custom...